Hospital/src/patterns/Hospital.java

```diff
@@ -0,0 +1,22 @@
+package patterns;
+
+public class Hospital {
+	
+	public static void main(String[] args){
```

Comment:
[High] This *main* do nothing. It just creates rooms but you're not using all these variables (cuarto1, cuarto2, ...,cuarto6)

The purpose of this exercise is to create an HospitalFactory, where depending on a Hospital type (it could be small and big hospital, or different kind of hospital), you get a specific Hospital building.